shared/profile/user/index.js

```diff
@@ -250,6 +252,17 @@ class BioTeamProofs extends React.PureComponent<BioTeamProofsProps> {
           </Kb.Text>
           <Teams username={this.props.username} />
           <Proofs {...this.props} />
+          {flags.proofProviders && (
+            <Kb.Box2 direction="horizontal" style={styles.addIdentityContainer}>
```

Comment:
this block should only show up given some other kind of prop. as 1. it should only show if there are additional identities and 2. if should only show on your own profile, so we at least need `props.showOtherIdentities` or something

Reply:
:white_check_mark: